notebooks/LST1_observation_simulator.ipynb

```diff
@@ -0,0 +1,1013 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    from lstchain import version
</pre><p>maybe more explicit: import lstchain, then later lstchain.__version__</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Changed to
from lstchain import version as lstchain_version